### DISCUSSION TITLE
All Security Advisories have disappeared

### DISCUSSION THREAD

**Discussion by 9001** [community/community#102522] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

All security advisories, across all projects, appear to have become unlisted?

The Security tab has a numerical indicator showing the correct number of public advisories, but the actual page says "There aren’t any published security advisories"

some examples:
* https://github.com/9001/copyparty/security
* https://github.com/pallets/jinja/security

**Comment by 0xdea** [comment]
I'm experiencing the same problem. The URL of advisories (if you know them) still work, but advisories are not listed anymore. To the best of my knowledge, this happened during the weekend.

**Reply by LPardue** [reply]
Same for me, I noticed it occurring Friday evening UTC

**Comment by katesaikishore** [comment]
Same here

**Comment by KateCatlin** [comment, marked as answer]
Hi all! Thank you for flagging this. We're aware of the issue and are working on a fix, it should be out shortly. I'll respond back in this discussion when it is out!

**Comment by KateCatlin** [comment]
Thank you so much for your patience here! This issue has now been resolved. Again, we appreciate our community for making us aware ❤️

**Reply by 0xdea** [reply]
Thanks for the update and the quick fix!
